server.go

```diff
@@ -1944,6 +1963,59 @@ func SendHeader(ctx context.Context, md metadata.MD) error {
 	return nil
 }
 
+// SetSendCompressor sets a compressor for outbound messages.
+// It must not be called after any event that causes headers to be sent
+// (see _ServerStream_.SetHeader for a complete list). Provided compressor is used when below
+// conditions are met:
+//
+//   - compressor is registered via encoding.RegisterCompressor
+//   - compressor name exists in the client advertised compressor names sent in
+//     grpc-accept-encoding header. Use _ServerStream_.ClientAdvertisedCompressors
+//     to get client advertised compressor names.
+//
+// The context provided must be the context passed to the server's handler.
+// It must be noted that compressor name "identity" disables the outbound compression.
```

Comment:
Please use `encoding.Identity` instead to encourage the use of a defined constant over a direct string.